tracer/src/Datadog.Trace/Util/RuntimeId.cs

```diff
@@ -0,0 +1,60 @@
+// <copyright file="RuntimeId.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Util
+{
+    internal static class RuntimeId
+    {
+#if NETFRAMEWORK
+        private const string NativeLoaderLibName = "Datadog.AutoInstrumentation.NativeLoader.dll";
+#else
+        private const string NativeLoaderLibName = "Datadog.AutoInstrumentation.NativeLoader";
+#endif
```

Comment:
We need to be careful with this, can lead to issues with different bitness and platforms. That's the reason we rewrite PInvokes in the tracer, to ensure we are doing this right regardless of the platform. See: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp#L358-L440

Reply:
maybe worth to move that implementation to `shared`

Reply:
> We need to be careful with this, can lead to issues with different bitness and platforms. That's the reason we rewrite PInvokes in the tracer, to ensure we are doing this right regardless of the arch and platform. See: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp#L358-L440

Not sure to understand the issue at stake and what you would expect me to do.
Do you mean that we should add the bitness in the name of the library ?

Reply:
As discussed offline, for now, leave it like this.

Reply:
moving this to `shared` is not that simple. We would have to move the logging (interface and maybe the vendor(s)).